### PR TITLE
Add RReLU(Randomized Leaky ReLU)

### PR DESCRIPTION
**Pull request by kemaswill**
Add a new advanced activation: RReLU(Randomized Leaky ReLU), please refer to [Empirical Evaluation of Rectified Activations in Convolutional Network](https://arxiv.org/abs/1505.00853) for more details.

Similar to Dropout, RReLU use different negative slope coefficients in training and testing phrase respectively. During training, RReLU uses a random negative slope coefficients sampled from [l, u) while using the a fixed one (l+u)/2 during testing.

On the mnist_cnn.py example, using RReLU and ReLU can get accuracy of 98.61% and 98.57% respectively after 12 epochs.

I'm glad to add test cases if needed even though several current test cases fails due to unknown reasons.

### CODE REVIEW

**Comment by fchollet** [comment]
We will not be adding new APIs to core Keras for the time being, sorry. This layer would be a good fit for a "contrib" or "extras" repo instead. Please check with @patyork about the status of the contrib repo.

**Comment by kemaswill** [comment]
@patyork Could you tell me what's the current status of the contrib repo? Thanks a lot.

**Comment by patyork** [comment]
The contribution repo is being discussed pretty thoroughly in [this issue](https://github.com/fchollet/keras/issues/4944); I don't think it is available as of yet.

**Comment by tboquet** [comment]
@kemaswill the repo is ready you could open a PR [here](https://github.com/farizrahman4u/keras-contrib) :tada: !
